### DISCUSSION TITLE
go build 构建失败

### DISCUSSION THREAD

**Discussion by fsy2020** [coaidev/coai#279] [Q&A | 问题解答] [answered]
docker运行成功。没写过go，这个是文件缺失的bug吗
![image](https://github.com/user-attachments/assets/34127ff0-9240-4e1e-935d-4338aca54bfa)

[a comment by an automated account is not shown]

**Comment by imszy** [comment, marked as answer]
1. execute command:
> sudo apt-get update
> sudo apt-get install libwebp-dev


2. enable Cgo:
> export CGO_ENABLED=1


3. check if enabled or not:
> go env CGO_ENABLED

enabled if return 1

[a comment by an automated account is not shown]
